x/wasm/types/validation.go

```diff
@@ -5,7 +5,10 @@ import (
 )
 
 var (
-	// MaxLabelSize is the longest label that can be used when Instantiating a contract
+	// MaxSaltSize is the longest label that can be used when instantiating a contract
+	MaxSaltSize = 128 // extension point for chains to customize via compile flag.
```

Comment:
The way the length encoding is specified does not allow value > 255. So I don't think this should be configurable.